### DISCUSSION TITLE
Incorporate Honeyd for multiple honeypots on the same instance

### DISCUSSION THREAD

**Discussion by alphaDev23** [telekom-security/tpotce#1019] [Ideas]
While honeyd is dated, it still works and allows for multiple honeypots on the same instance via configuration.  https://github.com/DataSoft/Honeyd.  Additionally, it allows for additional low interaction services via python scripting.  To connect an attacker to T-Pot's existing individualized honeypots simply proxy the connection. 

This would allow users to intermix real servers with numerous honeypots.

**Comment by t3chn0m4g3** [comment]
Thanks for the idea (updated your [link](https://github.com/DataSoft/Honeyd), though).

While HoneyD might still be a great honeypot and checking all the boxes its project has been abandoned. Neither the issues, nor the PRs are being actively managed, not to mention its codebase.

So, when we started with T-Pot back in **2015** we already decided against HoneyD for the mentioned reasons. Ever since we have deprecated honeypots that are not actively maintained. I.E. for T-Pot 22.03 **all** Py2 based honeypots using deprecated Py2 modules via PyPi have already been removed and have been replaced with new honeypots. This is part of our due diligence.

Thanks again and sorry, but no, we will not incorporate HoneyD.

**Reply by alphaDev23** [reply]
Will it be possibly to create multiple honeypots from one instance on the same subnet with T-Pot in the future?

**Reply by t3chn0m4g3** [reply]
Please elaborate.

**Comment by alphaDev23** [comment]
See the functionality, not the implementation, of HoneyD.  This would allow users to intermix real servers with numerous honeypots.
